src/api/methods/getUnspent.js

```diff
@@ -3,11 +3,35 @@ const { omit } = require('lodash');
 const getColor = require('./getColor');
 const { filterUnspent } = require('../../utils');
 
+function othersHeartbeatUtxos(bridgeState) {
+  if (
+    bridgeState.config.heartbeat &&
+    bridgeState.config.heartbeat.filter &&
+    bridgeState.account
+  ) {
+    const {
+      currentState: { unspent },
+      account: { address },
+      config: {
+        heartbeat: { color },
+      },
+    } = bridgeState;
+    return Object.keys(unspent).filter(
+      k =>
```

Comment:
We can export two methods from `filterUnspent` utility from [here](https://github.com/leapdao/leap-node/blob/60be45777457c47af53b43bef9f2b2424ea7af2b/src/utils/filterUnspent.js#L8) and then filter unspents in the [same fashion](https://github.com/leapdao/leap-node/blob/60be45777457c47af53b43bef9f2b2424ea7af2b/src/utils/filterUnspent.js#L23) for consitency sake ;)

Reply:
I'd prefer to keep it like this if you don't mind. `filterUnspent` is used on other modules, and I don't think it's the right time to refactor :)

Reply:
Just a note ^__^